x-pack/plugins/monitoring/server/lib/alerts/fetch_nodes_from_cluster_stats.ts

```diff
@@ -87,7 +87,7 @@ export async function fetchNodesFromClusterStats(
                   },
                 ],
                 _source: {
-                  includes: ['cluster_state.nodes', 'elasticsearch.cluster.stats.nodes'],
+                  includes: ['cluster_state.nodes', 'elasticsearch.cluster.stats.states.nodes'],
```

Comment:
This was a bug. `nodes` are found in `stats.state`, not `state` https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-metricset-elasticsearch-cluster_stats.html